test/index.test.js

```diff
@@ -235,7 +235,8 @@ describe('Facebook Pixel', function() {
             price: 24.75,
             name: 'my product',
             category: 'cat 1',
-            sku: 'p-298'
+            sku: 'p-298',
+            value: 24.75
```

Comment:
@amnoonan could you update this to be something other than `24.75` so we can make this test more explicit? Otherwise, this test still doesn't tell me if I'm mapping `price` or `value`